scripts/clear-release-pr.mjs

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env node
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+
+// This script creates a release PR
+import { execSync, exec, spawn } from "child_process";
+import commandLineArgs from "command-line-args";
+import fse from "fs-extra";
+import { basename } from "path";
+import semver from "semver";
+import { promisify } from "util";
+
+const {
+  SemVer,
+  valid: semverValid,
+  rcompare: semverRcompare,
+  lte: semverLte,
+} = semver;
+const { readJsonSync } = fse;
+const execP = promisify(exec);
+
+const options = commandLineArgs([
+  {
+    name: "type",
+    defaultOption: true,
+  },
+  {
+    name: "preid",
+  },
+]);
+
+const validReleaseValues = [
+  "major",
+  "minor",
+  "patch",
+];
+const validPrereleaseValues = [
+  "premajor",
+  "preminor",
+  "prepatch",
+  "prerelease",
+];
+const validPreidValues = [
+  "alpha",
+  "beta",
+];
+
+const errorMessages = {
+  noReleaseType: `No release type provided. Valid options are: ${[...validReleaseValues, ...validPrereleaseValues].join(", ")}`,
+  invalidRelease: (invalid) => `Invalid release type was provided (value was "${invalid}"). Valid options are: ${[...validReleaseValues, ...validPrereleaseValues].join(", ")}`,
+  noPreid: `No preid was provided. Use '--preid' to specify. Valid options are: ${validPreidValues.join(", ")}`,
+  invalidPreid: (invalid) => `Invalid preid was provided (value was "${invalid}"). Valid options are: ${validPreidValues.join(", ")}`,
+  wrongCwd: "It looks like you are running this script from the 'scripts' directory. This script assumes it is run from the root of the git repo",
+};
+
+if (!options.type) {
+  console.error(errorMessages.noReleaseType);
+  process.exit(1);
+}
+
+if (validReleaseValues.includes(options.type)) {
+  // do nothing, is valid
+} else if (validPrereleaseValues.includes(options.type)) {
+  if (!options.preid) {
+    console.error(errorMessages.noPreid);
+    process.exit(1);
+  }
+
+  if (!validPreidValues.includes(options.preid)) {
+    console.error(errorMessages.invalidPreid(options.preid));
+    process.exit(1);
+  }
+} else {
+  console.error(errorMessages.invalidRelease(options.type));
+  process.exit(1);
+}
+
+if (basename(process.cwd()) === "scripts") {
```

Comment:
I didn't think it was possible. Will try.

Reply:
Okay so it is possible to make sure that we are at the root of some repo, but I don't know if it is worth it. I added this check because I thought that some people were try and `cd` into the scripts directory and then run this script and this is a warning against that.

Reply:
That's fair, this is just a tool for us